rust/toolchain.bzl

```diff
@@ -874,6 +874,7 @@ rust_toolchain = rule(
                 "For more details see: https://docs.bazel.build/versions/master/skylark/rules.html#configurations"
             ),
         ),
+        # TODO: #3115 - Remove this attribute.
```

Comment:
Why aren't we removing this one too? When should we / what's blocking doing so?

Reply:
AFAICT `toolchain.crosstool_files` is unused in our codebase, so we should be able to just drop it.